os/net/ipv6/uip-ds6-route.c

```diff
@@ -133,8 +133,8 @@ assert_nbr_routes_list_sane(void)
 /*---------------------------------------------------------------------------*/
 #if UIP_DS6_NOTIFICATIONS
 static void
-call_route_callback(int event, uip_ipaddr_t *route,
-		    uip_ipaddr_t *nexthop)
+call_route_callback(int event, const uip_ipaddr_t *route,
+		    const uip_ipaddr_t *nexthop)
```

Comment:
Could you fix the code style here? `const` should start under `int event` like so
```
call_route_callback(int event, const uip_ipaddr_t *route,
		    const uip_ipaddr_t *nexthop)
```